### DISCUSSION TITLE
Number of Total Projects is not showing on GitHub Navbar

### DISCUSSION THREAD

**Discussion by MusfiqDehan** [community/community#60058] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

Hello,

I have **3 open projects** and **1 closed project** in my profile. But in the navbar section, it is not showing the number of projects. I created these projects around one month ago. Still, it isn't showing the No of Total Projects like other sections. **Is it a bug in GitHub?** What is the issue? Can any kind soul answer it?

Below I am comparing **my profile** and the profile of **Eddie**.  Eddie has no projects but it is showing 2, I have three projects but it is showing none.

![image](https://github.com/community/community/assets/47440165/d718a159-8202-4ade-9e2c-3f640bf6cfd1)
![image](https://github.com/community/community/assets/47440165/08f0b97d-1cdd-4335-a18c-04fe36be642b)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
HI @MusfiqDehan, 

The number showing is the sum of public projects (classic) + public Projects projects, however, due to some permission changes they may show incorrect counts or nothing. This is currently expected behavior from our side, but we are tracking this internally. Thanks for raising this 🚀 .
